### PR TITLE
feat(md): added a default component if preview environments configured but don't exist

### PR DESCRIPTION
**Pull request by ranihorev**
image

The link to the docs is optional and will appear only if configured

### CODE REVIEW

File: packages/core/src/managed/overview/PreviewEnvironments.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  isConfigured?: boolean;
+}
+
+export const PreviewEnvironments = ({ orderedEnvironments, isConfigured }: IPreviewEnvironmentsProps) => {
+  const { environments: previewEnvironments, ...previewEnvironmentsProps } = orderedEnvironments;
+  const logClick = useLogEvent('PreviewEnvironments');
+  const docsLink = getDocsUrl('previewEnvironments');
+  return (
+    <>
+      {(Boolean(previewEnvironments.length) || isConfigured) && (
```
Comment (luispollo) [inline packages/core/src/managed/overview/PreviewEnvironments.tsx:20]: Just out of curiosity, what's the advantage of writing it this way versus `(isConfigured && previewEnvironments.length > 0)` (which is more readable IMHO)?

File: packages/core/src/managed/overview/PreviewEnvironments.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        <h4 className="sp-margin-2xl-top sp-margin-m-bottom self-left">
+          <b>Preview Environments</b>
+        </h4>
+      )}
+      {!Boolean(previewEnvironments.length) && isConfigured && (
```
Comment (luispollo) [inline packages/core/src/managed/overview/PreviewEnvironments.tsx:25]: Same comment/question here. I find this much harder to read than a simple greater-than comparison.

**Review by luispollo — APPROVED** [review]
Minor question inline, otherwise LGTM!

File: packages/core/src/managed/overview/PreviewEnvironments.tsx (hunk 1 above)
Comment (ranihorev) [reply packages/core/src/managed/overview/PreviewEnvironments.tsx:20]: No advantage, just a personal perference. I don't mind changing though :)

**Pull request merged**